getting_started/installation.md

```diff
@@ -23,6 +23,12 @@ Using PowerShell (Windows):
 irm https://deno.land/install.ps1 | iex
 ```
 
+Using [Winget](https://winget.run) (Windows):
+
+```shell
+winget install deno
```

Comment:
The winget registry page seems suggesting the command `winget install -e --id DenoLand.Deno` https://winget.run/pkg/DenoLand/Deno

Is this command reliable enough?